docs/release/updating.md

```diff
@@ -23,6 +23,9 @@ The recommended procedure for the upgrade is:
 
 - On all servers, admin nodes, and clients,
   perform the RPM update to the new DAOS fix level.
+  Set up the new DAOS repository, then run `yum update` (CentOS 7 or EL8),
```

Comment:
Is it worth adding a "!!! warning" note that if the yum/dnf "install" sub-command is used inadvertently (instead of update) that problems could occur?

Reply:
my personal opinion: no. if people don't read the text that explains which commands they should use, we'll have to put warnings everywhere. this is a pretty standard sysadmin operation and i don't feel that the DAOS guide needs to go into more details here.

Reply:
the problem is that an (apparently) knowledgeable person used 'yum install' instead of 'yum update' (and that sparked this whole change.